src/utils.js

```diff
@@ -275,15 +276,25 @@ const mergeErrors = (object, other) => {
   return mergeWith(object, other, customizer);
 };
 
-const redirectToProxyLogin = (enterpriseSlug) => {
+const getProxyLoginUrl = (enterpriseSlug) => {
```

Comment:
are these util functions something that should be moved to frontend-enterprise? i assume there is similar functionality in the learner portal

Reply:
We do compute the proxy login URL in the learner portal where these util function could be reused but I'm hesitant to consolidate in frontend-enterprise because the code duplication here is mostly negligible and IMO not worth the cost of having to upgrade dependencies in admin/learner portals anytime the logic changes in frontend-enterprise.

Plus, in a world where the admin/learner portals are consolidated in a single experience, much of the logic around redirecting to the proxy-login could be done consistently in one place across both admin + learners as opposed to implementing and sharing duplicate logic for each distinct user role / portal.